### DISCUSSION TITLE
Cannot open Source file stdio.h

### DISCUSSION THREAD

**Discussion by soky** [Mair/esp32-course#82] [Course Qeustions]
When starting a new project I get the error Cannot open source file   stdio.h
Error continues with consider updating your compile_commands.json  or  includepath

I am totally lost at this point. Any help is appreciated.

soky

**Comment by Mair** [comment]
@soky Could you please share a screen shot?

**Comment by soky** [comment]
I completely uninstalled VSC and reinstalled it. I will try again to see if the problem is solved.

**Comment by Mair** [comment]
@soky  The issue is most likely not with VSC. its more likely with your IDF installation or the esp IDF extension.
My guess is that its your IDF extension configuration. If your installation of VSC does not work then try uninstalling and re-installing the IDF extension and follow the videos to re-configure it
